internal/system/agent/v2/controller/http/controller.go

```diff
@@ -28,12 +29,29 @@ func NewAgentController(dic *di.Container) *AgentController {
 }
 
 func (c *AgentController) GetHealth(w http.ResponseWriter, r *http.Request) {
-	lc := container.LoggingClientFrom(c.dic.Get)
+	lc := bootstrapContainer.LoggingClientFrom(c.dic.Get)
 
 	vars := mux.Vars(r)
 	services := strings.Split(vars["services"], ",")
 
-	health := application.GetHealth(services, container.RegistryFrom(c.dic.Get))
+	health := direct.GetHealth(services, bootstrapContainer.RegistryFrom(c.dic.Get))
 	res := responses.NewHealthResponse("", "", http.StatusOK, health)
 	pkg.Encode(res, w, lc)
 }
+
+func (c *AgentController) GetMetrics(w http.ResponseWriter, r *http.Request) {
+	lc := bootstrapContainer.LoggingClientFrom(c.dic.Get)
+	ctx := r.Context()
+	vars := mux.Vars(r)
+	services := strings.Split(vars["services"], ",")
```

Comment:
"services" should be a constant, and reuse the CommaSeparator
https://github.com/edgexfoundry/go-mod-core-contracts/blob/2c76dd9b4bb8135d36e7dd88eb99c6fe0fcdb492/v2/constants.go#L171